client/state/selectors/get-checklist-task-urls.js

```diff
@@ -44,17 +42,11 @@ export default createSelector(
 		const contactPageUrl = getPageEditorUrl( state, siteId, getContactPage( posts ) );
 		const frontPageUrl = getFrontPageEditorUrl( state, siteId );
 
-		const updateHomepageUrl = isEnabled( 'checklist-homepage-template-select' )
-			? addQueryArgs( frontPageUrl || getEditorUrl( state, siteId, null, 'page' ), {
-					'new-homepage': 1,
-			  } )
-			: frontPageUrl;
-
 		return {
 			post_published: getPageEditorUrl( state, siteId, firstPostID ),
 			contact_page_updated: contactPageUrl,
 			about_text_updated: frontPageUrl,
-			front_page_updated: updateHomepageUrl,
+			front_page_updated: frontPageUrl,
```

Comment:
Should this line still retain the `|| getEditorUrl( state, siteId, null, 'page' )` fallback in cases where the site doesn't already have a page that is set to `page_on_front`? This is an edge case that might not be possible right now, so I could be overly cautious here — but I guess if we imagine a 'blank-slate' site that doesn't have any pages, then the checklist task for updating the homepage could wind up with an empty link in the task's CTA?